From 08f0fe9e1fe48baabeb74e642c3d38b54c552290 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:26 -0400
Subject: [redhat] kabi: Adding symbol skb_recv_datagram (net/core/datagram.c)

Message-id: <c6a152db3c54b88e3edb28b63b72d8530717b6ad.1410510773.git.poros@redhat.com>
Patchwork-id: 96138
O-Subject: [PATCH RHEL7.1 206/233] kabi: Adding symbol skb_recv_datagram (net/core/datagram.c)
Bugzilla: 1071099
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Standard UDP interface.

    We register a function with the OS called as llt_udp_recv(), which
    gets called when we have packets arriving on the LLT link on the
    configured UDP port. In this function we get pointer to "struct sock"
    as an argument. To get skbs (Linux Socket Buffers) from this socket,
    we call skb_recv_datagram() in a loop on the "sock" pointer and process
    the skb and repeat this operation until we can not get any more skbs
    from the socket pointer. - VCS team

Signed-off-by: Petr Oros <poros@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/skb_recv_datagram b/redhat/kabi/kabi-module/kabi_ppc64/skb_recv_datagram
new file mode 100644
index 0000000..1262240
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/skb_recv_datagram
@@ -0,0 +1,2 @@
+#1-
+0x17b75f21 skb_recv_datagram vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/skb_recv_datagram b/redhat/kabi/kabi-module/kabi_s390x/skb_recv_datagram
new file mode 100644
index 0000000..b1a51ce
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/skb_recv_datagram
@@ -0,0 +1,2 @@
+#1-
+0x48ef7243 skb_recv_datagram vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/skb_recv_datagram b/redhat/kabi/kabi-module/kabi_x86_64/skb_recv_datagram
new file mode 100644
index 0000000..81b09a6
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/skb_recv_datagram
@@ -0,0 +1,2 @@
+#1-
+0xd0a4c8a9 skb_recv_datagram vmlinux EXPORT_SYMBOL
-- 
1.7.1